.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+## To encourage contributors to use issue templates, we don't allow blank issues
+blank_issues_enabled: false
```

Comment:
not sure if disabling blank issues is too restrictive for this repo tho

Reply:
I think that's fine! Thanks @micalevisk